ee_tests/cico_run_EE_tests_ts.sh

```diff
@@ -7,6 +7,9 @@
 # $5 = quickstart name
 # $6 = release strategy
 # $7 = feature level
+# $8 = enable zabbix reporting
+# $9 = zabbix host
+# $10 = zabbix metric prefix
```

Comment:
We agreed that we wouldn't introduce new property on top-level for zabbix prefix, we would just reuse feature_level